components/Button/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import styles from './index.module.scss';
+import ButtonChildren from './ButtonChildren';
+
+type DeviceType = 'desktop' | 'tablet' | 'mobile';
+type ButtonType =
+  | 'login'
+  | 'primary'
+  | 'secondary'
+  | 'delete'
+  | 'add-column'
+  | 'add-todo'
+  | 'delete-dashboard'
+  | 'pagenation'
+  | 'add-board'
+  | 'dashboard';
+
+interface ButtonProps {
+  deviceType: DeviceType;
+  buttonType: ButtonType;
+  isOwner?: boolean;
+  disable?: boolean;
+  onClick?: (event: React.MouseEvent<HTMLButtonElement>) => void;
+  children?: string;
+}
+
+const Button: React.FC<ButtonProps> = ({
+  deviceType,
+  buttonType,
+  isOwner,
+  disable = false,
+  onClick,
+  children,
+}) => {

```

Comment:
저번에 간략히..! 컨벤션에 대해 말씀드렸지만 컴포넌트는 `function 컴포넌트이름 () {}` 으로 하기로 해서 수정 가능하실까요..?!
이런식으로 React.FC 사용 안해주시면 좋을거같아요!!
```suggestion
function Button({
  deviceType,
  buttonType,
  isOwner,
  disable = false,
  onClick,
  children,
} : ButtonProps) {
```